source/strings.tex

```diff
@@ -1478,32 +1444,19 @@
 \begin{itemdescr}
 \pnum
 \effects
-If
-\tcode{*this}
-and \tcode{str} are not the same object,
-modifies
-\tcode{*this}
-as shown in Table~\ref{tab:strings.op=}.
-
-\pnum
-If
-\tcode{*this}
-and \tcode{str} are the same object,
-the member has no effect.
+If \tcode{*this} and \tcode{str} are not the same object, modifies \tcode{*this}.
```

Comment:
I think we still need to say "as described below" or something here. Otherwise, a reader is left thinking "ok, but how?" Maybe this should be merged into the postconditions instead, though, since the "points at [...] an allocated copy" postcondition is not correct in the `this == &str` case.

Reply:
I turned the two conditions around and made the second one say "see below".